schemaregistry/serde/jsonschema/json_schema_util.go

```diff
@@ -238,6 +255,9 @@ func disjoint(slice1 []string, map1 map[string]bool) bool {
 }
 
 func getField(msg *reflect.Value, name string) (*reflect.Value, error) {
+	if msg.Kind() != reflect.Struct {
+		return nil, fmt.Errorf("message is not a struct")
```

Comment:
why not do the same thing as above
`val.MapIndex(reflect.ValueOf(propName)`

Reply:
Good question.  This method is only for operating on Structs